config.example.wikimedia.yaml

```diff
@@ -184,6 +196,97 @@ templates:
 #              service: 
 #                uri: http://revscore.wikimedia.org/{domain}/{title}/{revision}
 
+  global-sys: &gb/sys/1.0.0
+    info:
+      title: global domain sys API module
+      version: 1.0.0
+    paths:
+      /{module:table}: *wp/sys/table
+      /{module:key_value}: *wp/sys/key_value
+      /{module:post_data}: *wp/sys/post_data
+      /{module:mathoid}/v1/request/{format}:
```

Comment:
@Pchelolo the security stanza has been added to the POST public endpoint in `spec/media/v1/mathoid.yaml`